scripts/windows-2019-provision.ps1

```diff
@@ -215,3 +250,23 @@ Select-Object -Property DeviceID, DriveType, VolumeName,
 
 Write-Host "Patch(s) installed"
 Get-HotFix | Format-Table -Property HotFixID, Description, InstalledOn
+
+Write-Host "== Sanity Check of installed tools"
+& docker -v ## Client only
+
+& "$baseDir\git\cmd\git.exe" --version
+& "$baseDir\jdk-8\bin\java.exe" -version
+& "$baseDir\jdk-11\bin\java.exe" -version
+& "$baseDir\jdk-17\bin\java.exe" -version
+& "$baseDir\container-structure-test.exe" version
+& "$baseDir\hadolint.exe" --version
+& "$baseDir\docker-compose.exe" --version
+& "$baseDir\git\mingw64\bin\git-lfs.exe" --version
+& "$baseDir\jx-release-version" --version
+& "$baseDir\jq.exe" --version

```

Comment:
Is the $baseDir added to the $PATH? If so, shouldn't the sanity checks run without the folder to ensure they're available everywhere?

Reply:
That is the pattern with shell script yes.

But with powershell, the PATH is written in the local register, and is read only after the VM reboot if I understand correctly.

There might be a technique that I don't know about, but even when I failed with trying to spawn either a new `cmd` or `powershell` subprocess, so decided to go that way: at least it allowed be to discover issues on the current Windows templates :)